checks/evaluation/permissions/permissions.go

```diff
@@ -49,7 +101,18 @@ func TokenPermissions(name string, c *checker.CheckRequest, r *checker.TokenPerm
 		return checker.CreateInconclusiveResult(name, "no github tokens found")
 	}
 
-	score, err := applyScorePolicy(r, c)
+	if len(r.TokenPermissions) == 0 {
+		text := "no workflows found in the repository"
+		return checker.CreateInconclusiveResult(name, text)
+	}
```

Comment:
any idea if/how this differs from the early exit above?

Reply:
This was unnecessary. Sorry for that. Removed in https://github.com/ossf/scorecard/pull/3693/commits/d49d788ac83c09e32c59bd43d499c6599ca050eb